.github/workflows/build.yml

```diff
@@ -247,7 +247,7 @@ jobs:
         uses: actions/upload-artifact@b4b15b8c7c6ac21ea08fcf65892d2ee8f75cf882 # v4.4.3
         with:
           name: com.x8bit.bitwarden.aab
-          path: app/build/outputs/bundle/standardRelease/com.x8bit.bitwarden-standard-release.aab
+          path: app/build/outputs/bundle/standardRelease/com.x8bit.bitwarden-standard.aab
```

Comment:
Won't this add confusion on what variant the file actually is?

Reply:
Absence of a build type designation will imply it's the "release" version. All other build types include a suffix to indicate the build type.

```
standardRelease -> com.x8bit.bitwarden.apk
standardBeta    -> com.x8bit.bitwarden.beta.apk
standardDebug   -> com.x8bit.bitwarden.dev.apk

fdroidRelease   -> com.x8bit.bitwarden-fdroid.apk
fdroidBeta      -> com.x8bit.bitwarden.beta-fdroid.apk
fdroidDebug     -> com.x8bit.bitwarden.dev-fdroid.apk
```